src/averageAges.js

```diff
@@ -15,11 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let menInCentury = {};
+
+  if (arguments.length === 1) {
+    menInCentury = people.filter(
+      person => person.sex === 'm');
+  } else {
+    menInCentury = people.filter(
+      person => person.sex === 'm' && (
+        Math.ceil(person.died / 100) === century));
+  }
```

Comment:
you can simplify your code just using ternary operator. Apply for other functions
Example:
`const men = people.filter(person => century ? person.sex === 'm' && Math.ceil(person.died / 100) === century : person.sex === 'm')`